README.md

```diff
@@ -164,6 +164,9 @@ Under the `azure` section, edit following values as per your configuration
 * `numnodes` to change the cluster size in terms of number of nodes deployed
 * `vm_sku` to specify the VM size to use. You can choose from the
   [available VM sizes](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-general).
+* `use_adlsg2` to use Azure Data Lake Storage(ADLS) Gen2 as datastore for Accumulo
```

Comment:
When a user sets this to true, would there be any cleanup they need to do after shutting a cluster down when ADLS is automatically setup?

Reply:
Yes, cleanup is required. This will be implemented for both “wipe” and “terminate” actions of muchos command.  Due to dependency, we will push a separate PR for “wipe” and “terminate” actions once this PR is approved.
Note: The terminate action is currently unsupported for Azure based clusters

Reply:
Would it make sense to open issues for this follow on work?

Reply:
We will open PR directly because the work is already complete and just need to upstream it once this PR is approved